lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestContextStack.cs

```diff
@@ -0,0 +1,226 @@
+            {
+                Logger.Log(Level.Info, "ContextId: " + value.Id);
+                switch (value.Id)
+                {
+                    case ContextOneId:
+                        var contextConfig =
+                            Common.Context.ContextConfiguration.ConfigurationModule.Set(
+                                Common.Context.ContextConfiguration.Identifier, ContextTwoId)
+                                .Build();
+                        var stackingContextConfig =
+                            TangFactory.GetTang()
+                                .NewConfigurationBuilder()
+                                .BindImplementation(GenericType<IInjectableInterface>.Class,
+                                    GenericType<InjectableInterfaceImpl>.Class)
+                                    .Build();
+
+                        value.SubmitContext(Configurations.Merge(stackingContextConfig, contextConfig));
+                        break;
+                    case ContextTwoId:
+                        value.SubmitTask(
+                            TaskConfiguration.ConfigurationModule.Set(TaskConfiguration.Identifier, "contextStackTestTask")
+                                .Set(TaskConfiguration.Task, GenericType<TestContextStackTask>.Class)
+                                .Build());
+                        break;
+                    default:
+                        throw new Exception("Unexpected ContextId: " + value.Id);
+                }
+            }
+
+            public void OnNext(ICompletedTask value)
+            {
+                Logger.Log(Level.Info, TaskValidationMessage);
+                value.ActiveContext.Dispose();
+            }
+
+            public void OnNext(IClosedContext value)
+            {
+                Logger.Log(Level.Info, ClosedContextValidationMessage);
+                if (value.ParentContext != null)
+                {
+                    value.ParentContext.Dispose();
+                }
+            }
+
+            public void OnNext(IRunningTask value)
+            {
+                Logger.Log(Level.Info, "Test task is running.");
+            }
+
+            public void OnNext(ICompletedEvaluator value)
+            {
+                Logger.Log(Level.Info, "Evaluator completed.");
+            }
+
+            public void OnNext(IFailedTask value)
+            {
+                throw new Exception("Task not expected to fail.");
+            }
+
+            public void OnError(Exception error)
+            {
+                throw new NotImplementedException();
+            }
+
+            public void OnCompleted()
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        /// <summary>
+        /// A Task to ensure that an object configured in the second context configuration 
+        /// is properly injected.
+        /// </summary>
+        private sealed class TestContextStackTask : ITask
```

Comment:
I think we should make an exception for these classes, since:

1. These classes already exist in other tests.
2. It clutters up the project if we were to add more of these just for testing purposes.